2-mandatory/1-writers.js

```diff
@@ -72,6 +77,13 @@ Exercise 2:
 
 function logDeadWritersInTheirForties() {
   // write your code here
+  writers.forEach((element) => {
+    if (element.alive === false && element.age > 40 && element.age < 50) {
```

Comment:
Can you think of another Javascript API that could be used instead of having an `if` statement?